public/configs/v1/env.json

```diff
@@ -291,8 +288,8 @@
          "endpoints": {
             "indexers": [
                {
-                  "api": "https://indexer.v4dev.dydx.exchange",
-                  "socket": "wss://indexer.v4dev.dydx.exchange"
+                  "api": "https://indexerv4dev.dydx.exchange",
```

Comment:
Did we mean to change this?

Reply:
Yes, updated this. Since `https://indexerv4dev.dydx.exchange` gets processed through cloudflare and `https://indexer.v4dev.dydx.exchange` does not, it is better to use `https://indexer.v4dev.dydx.exchange`

Reply:
oo gotcha. i wonder if there's a world where we can convert this to a js/ts file so we can add comments 🤔 that would be super nice